onboarding.md

```diff
@@ -108,8 +108,15 @@ The project has a venue for real-time discussion:
   * This is a repository to which all members of the `nodejs` GitHub
     organization (not just collaborators on Node.js core) have access. Its
     contents should not be shared externally.
+  * Node.js has a moderation team which you should contact when unsure
```

Comment:
This is a good place to mention the existence of the `@nodejs/moderation` team here.